support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -289,10 +289,8 @@ function onPaymentAuthorised(
 				productType,
 			);
 
-			const inThreeTierVariant = inThreeTierV2Variant(
-				state.common.abParticipations,
-			);
-			if (inThreeTierVariant) {
+			const inThreeTier = showThreeTierCheckout(state.common.abParticipations);
```

Comment:
We're calling `showThreeTierCheckout(state.common.abParticipations)` in 2 places in this file, we could call it once and assign the result to a variable in this case to save calling it multiple times?